src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc.csproj

```diff
@@ -1,6 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
```

Comment:
Any reason to remove this comment?

Reply:
(I thought this was added by some tool automatically.)
This comment was moved from the .csproj files to Common.prod.props. 
And since there's only 1 TFM here, there's no ordering concern.